2022/wot-cg-2022.html

```diff
@@ -0,0 +1,297 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Web of Things Community Group Charter
+    </title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="//www.w3.org/StyleSheets/TR/base">
+    <style>
+      body {
+        max-width: 60em;
+        margin: auto;;
+      }
+      *:target {
+        background-color: yellow;
+      }
+      li {
+        margin-bottom: 9pt;
+      }
+      .note {
+        background-color: yellow;
+        padding: 10px;
+      }
+      .remove {
+        background-color: yellow;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>
+      [DRAFT] Web of Things Community Group Charter
+    </h1>
+    <p>
+      <span class="remove">{TBD: remove next sentence before submitting for
+      approval}</span> This Charter is work in progress. To submit feedback,
+      please use <span class="remove">https://github.com/w3c/charter-drafts/issues</span> Issues or 
+      the <span class="remove">https://github.com/w3c/charter-drafts/pull/392</span> Pull Requst 392
+      where Charter is being developed.
+    </p>
+    <ul>
+      <li>This Charter: <span class="remove">https://github.com/w3c/charter-drafts/blob/gh-pages/2022/wot-cg-2022.html</span>
+      </li>
+      <li>Previous Charter: <span class="remove">https://www.w3.org/community/wot/charter/</span>
+      </li>
+      <li>Start Date: <span class="remove">01 June 2022 (tentative)</span>
+      </li>
+      <li>Last Modifed: <span class="remove">03 May 2022</span>
+      </li>
+    </ul>
+    <h2 id="goals">
+      Goals
+    </h2>
+    <p>
+      The goal of WoT Community Group is to accelerate the community activities around the recommendations and notes published by the WoT WG and IG. More concretely, the group aims to:
+      <ul>
+        <li>Increase awareness around the WoT specifications</li>
+        <li>Collect use cases from the wider community</li>
+        <li>Collect implementation experience and technologies from the wider community</li>
+        <li>Facilitate the implementation of the specifications by providing help in form of guidelines and tutorials</li>
+      </ul>
+    </p>
+    <h2 id="scope-of-work">
+      Scope of Work
+    </h2>
+    <p>
+      In order to achieve the above goals, the scope of the Web of Things Community Group includes the following activities:
+      <ul>
+        <li>Organizing events such as hackathons, ideathons, etc.</li>
```

Comment:
Hackathons seem to be specifically competitive in nature.  I think we should at least consider the option of whether plugfests should be a CG activity, especially since we may want to use them to engage with non-IG members.

```suggestion
        <li>Organizing events such as hackathons, plugfests, ideathons, etc.</li>
```

Reply:
Plugfests are very welcomed, but we thought to keep them outside because of the possible conflicts with WG/IG maybe we should clarify that also plugfests will be in the scope of https://github.com/w3c/wot-cg/blob/main/wot-ig-collaboration.md

Reply:
I think the hackathon can be still used for non competitive occasions and I find it a "good word" since people know what it means, i.e. sit together for some short and build something. If some people sit together with some raspberry pis and build WoT devices, it would be a hackathon for me. Plugfest would be something that starts from that point onwards, coming together with already working WoT devices and checking how well they work together. Regarding moving plugfests to CG or not, I agree with @relu91 that it should be maintained in the document he linked.